backend/app/services/api/filterer.rb

```diff
@@ -38,7 +38,8 @@ def apply_full_text_filter
       associated_columns = Array.wrap(FULL_TEXT_EXTRA_TABLES[query.klass]).each_with_object({}) do |relation, res|
         res[relation] = localized_columns_for relation.to_s.classify.constantize
       end
-      self.query = query.where id: query.dynamic_search(localized_columns_for(query.klass), filters[:full_text], associated_columns).pluck(:id)
+      ids = query.klass.dynamic_search(localized_columns_for(query.klass), filters[:full_text], associated_columns).pluck(:id)
```

Comment:
ransack and pg_search have problem to work together so I have split query to two separate ones ;)

Reply:
Good to know.

Reply:
It might be possible to use subquery instead of plucking ids? Just in case somebody types, like, "e", and that matches the whole table worth of ids :), maybe it could work almost exactly like you have it, but using select instead of pluck:
fts_subquery = query.klass.dynamic_search(localized_columns_for(query.klass), filters[:full_text], associated_columns).select(:id)

self.query = query.where id: fts_subquery

Reply:
Done, thanks @agnessa ;). I have tried to break query to two by pluck, because that sql is huge and quite complex, but using select has its own advantages, so why not to go for it.